package.json

```diff
@@ -29,17 +29,17 @@
   },
   "dependencies": {
     "adm-zip": "^0.4.7",
-    "appium-adb": "2.0.0-beta14",
+    "appium-adb": "2.0.0-beta15",
```

Comment:
Yeah anything:
-beta -> change it to: beta
0.x.x -> change it  to latest or upgrade the deps to 1.0.0 (SemVer is not enable for 0.x.x packages.)